### ISSUE TITLE
Wonders of night sky

### ISSUE THREAD

**Issue by alex-w** [Stellarium/stellarium#499]
_Request from Kari Karjalainen in PM_

Do you have different tables in the planetarium, is there any plans to add a table with the names of the constellations and objects in each constellation - stars, deep sky.

I'm talking about the guide to the constellations, where all the information on the constellations is collected in the form of a table. People do not always know which objects and how many types of objects in each constellation. And by including such an atlas program, you can see which objects and what type (how many objects of each type) there are in each constellation.

**Comment by gzotti** [comment]
Have you tried what AstroCalc (F10) has in its What's up Tonight tab? 

For a purely static list I recommend one of the many textbooks.

**Comment by alex-w** [comment]
This is feature request to get the list of wonders per constellation, e.g. select Andromeda and get the list of interesting objects in the constellation.

**Comment by gzotti** [comment]
Yes, and just that info already exists in many printed "Guide to the Night Sky" books. Of course, another filter in WUT could select objects per constellation. Or yet another full tab with three columns: Constellation/object type (including "any")/objects.

**Comment by alex-w** [comment]
This is a good task for the community to participate in the contribution into Stellarium. Who wants to help us?
